### PR TITLE
Minor improvements to R11

### PR DESCRIPTION
**Pull request by AerialMantis**
**Summary**

* Change remaining spellings of `Sender` to `sender`.
* Rename `execution::scheduler` to `executor::schedule`.
* Move wording for the `submit` member function to the `sender` concept
definition.

**Todo**

* I believe there is still more to be done to the wording of the `submit` member function, this change is largely to reflect the change to the `executor` concept.

### CODE REVIEW

File: wording.md [hunk 1]
```diff
@@ -365,6 +365,24 @@ concept sender =
...
+
+In the Table below, 
+
+- `s` denotes a (possibly const) executor object of type `S`,
+- `cr` denotes the function or receiver object `DECAY_COPY(std::forward<R>(r))` 
+- `r` denotes a function or receiver object of type `R&&` invocable as `cr()` and where `decay_t<R>` models `move_constructible`.
+
+| Expression | Return Type | Operational semantics |
+|------------|-------------|---------------------- |
+| `s.submit(r)` | `void` | Evaluates `DECAY_COPY(std::forward<R>(r))` on the calling thread to create `cr` that will be invoked at most once by an execution agent. <br/> May block pending completion of this invocation. <br/> Synchronizes with [intro.multithread] the invocation of `r`. <br/>Shall not propagate any exception thrown by the function object or any other function submitted to the executor. [*Note:* The treatment of exceptions thrown by one-way submitted functions is described by the `execution::oneway_exception_handler` property. The forward progress guarantee of the associated execution agent(s) is implementation defined. *--end note.*] |
```
Comment (AerialMantis) [inline wording.md]: Currently, this matches the wording for the `executor` concept definition, is this the right place to introduce the contract for one of `value`, `error` or `done` being called on `r`?

Comment (kirkshoop) [reply wording.md]: I am not sure. I would go ahead and add it here, maybe?

Comment (jaredhoberock) [reply wording.md]: I put a first cut at defining the contract in the `jared-edits` branch as p2 of `execution::submit`'s specification.

Comment (AerialMantis) [reply wording.md]: Thanks Jared!

File: wording.md [hunk 2]
```diff
@@ -365,6 +365,24 @@ concept sender =
   sender-to-impl<S, sink_receiver>;
 ```
 
+None of a sender's copy constructor, destructor, equality comparison, or `swap` operation shall exit via an exception.
```
[the remark this answers is not in the document]
Comment (AerialMantis) [reply wording.md]: Yeah I wasn’t sure if all the same requirements that we have for executor should apply for sender, I was just being conservative.

I’ll take this part out.

**Comment by jaredhoberock** [comment]
Is this PR ready to merge?

**Comment by AerialMantis** [comment]
Yeah, I think this is good to merge now, I'll just push an update to resolve the conflicts with R11.

**Comment by jaredhoberock** [comment]
Thanks!

**Pull request merged**